build.gradle

```diff
@@ -70,4 +70,14 @@ subprojects {
       }
     }
   }
+
+  // Android dependency has different version for the compile and runtime
+  project.configurations.all {
```

Comment:
This is fine for now, but I think there was a reason we had to use newer builds tools than this. If this is introduced by Matt's dependency, then it needs to be fixed there so that this can be removed.